src/sentry/runner/commands/devserver.py

```diff
@@ -134,11 +137,10 @@ def devserver(
         bind = "127.0.0.1:8000"
 
     if ":" in bind:
-        host, port = bind.split(":", 1)
-        port = int(port)
+        host, port_s = bind.split(":", 1)
+        port = int(port_s)
     else:
-        host = bind
-        port = None
```

Comment:
Assuming this was the issue that resulted in:

```
    port = port + 1
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int
```

One option would be to add a default port number (i.e. 8000) but I'm guessing this is an unlikely edge case so not worth worrying about unless folks say otherwise.